MetaMask/metamask-extension
# feat: UX - Multichain - Disconnect All Modal

## **Description**

Creates a simple Disconnect All modal which can be used for both Accounts and Snaps

## **Related issues**

Fixes: [2212](https://app.zenhub.com/workspaces/metamask-wallet-ux-63529dce65cbb100265a3842/issues/gh/metamask/metamask-planning/2212)

Fixes: [2212](https://github.com/MetaMask/MetaMask-planning/issues/2212)

## **Manual testing steps**

1. Go to storybook
2. Check the Disconnect all Modal design is same as figma

## **Screenshots/Recordings**


### **Before**

NA
### **After**

<img width="339" alt="SCR-20240217-lmpr" src="https://github.com/MetaMask/metamask-extension/assets/46655/18a09e75-78f9-428b-b679-46348228db58">


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
